### PR TITLE
Add changelog items for the third version of kubeadm

### PR DESCRIPTION
**Pull request by luxas**
@mikedanese PTAL and merge, here are the release notes I aggregated for the third stable version of kubeadm

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
...
+ - The control plane version automatically defaults to the latest stable version of kubernetes [#37222](https://github.com/kubernetes/kubernetes/pull/37222)
+ - Use `etcd-3.0.14` as the default etcd version [#35921](https://github.com/kubernetes/kubernetes/pull/35921)
+ - Added a new command: `kubeadm token generate` [#35144](https://github.com/kubernetes/kubernetes/pull/35144)
+ - `kubeadm reset` drains the node it's run on and defaults to removing the node from the API Server as well [#37831](https://github.com/kubernetes/kubernetes/pull/37831)
+ - Updated the DNS addon to the v1.5 version [#37835](https://github.com/kubernetes/kubernetes/pull/37835)
+ - `kubeadm` now auto-starts the `kubelet` systemd service if inactive [#37568](https://github.com/kubernetes/kubernetes/pull/37568)
+ - Made the console output of `kubeadm` much cleaner and user-friendly [#37568](https://github.com/kubernetes/kubernetes/pull/37568)
+ - `kubectl logs` and `kubectl exec` can now be used with `kubeadm` clusters [TODOkubeletPR]
```
Comment (kad) [inline CHANGELOG.md]: this reference should be fixed before merging, right ?

Comment (luxas) [inline CHANGELOG.md]: Oops, forgot that one, fixed now

**Comment by luxas** [comment]
Have we hooked up the submit queue with this repo yet?

**Comment by mikedanese** [comment]
No, it's not.

**Comment by luxas** [comment]
Ok, merging manually then

**Pull request merged**